source/common/quic/envoy_quic_server_connection.h

```diff
@@ -107,6 +114,14 @@ class QuicListenerFilterManagerImpl : public Network::QuicListenerFilterManager,
       }
     }
   }
+  void onFirstPacketReceived(const quic::QuicReceivedPacket& packet) override {
+    for (auto& accept_filter : accept_filters_) {
+      Network::FilterStatus status = accept_filter->onFirstPacketReceived(packet);
+      if (status == Network::FilterStatus::StopIteration) {
```

Comment:
It looks lines onPeerAddressChanged also checks `connection.state() != Network::Connection::State::Open`. Should we check this too?

Reply:
I think that's specific to onPeerAddressChanged, which might actually close the connection. We don't pass the connection into onFirstPacketReceived anyway.